app/validators/seats_validator.rb

```diff
@@ -0,0 +1,17 @@
+class SeatsValidator
+  attr_reader :errors
+
+  def self.validate!(screening:, seats:)
+    return false if seats.blank?
```

Comment:
Validations should return errors for error handling or nothing if it passes, not true or false
```suggestion
    errors.add(:seat, :cannot_be_blank) and return if seats.blank?
```